modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -14,90 +14,57 @@ include::partial$acid-transactions-attributes.adoc[]
 
 include::{version-server}@sdk:shared:partial$acid-transactions.adoc[tag=intro]
 
-You may also want to start with our https://github.com/couchbaselabs/couchbase-transactions-java-examples[transactions examples repository],
-which features useful downloadable examples of using Distributed Transactions.
-
-Javadocs are available https://docs.couchbase.com/sdk-api/couchbase-transactions-java/index.html?overview-summary.html[online].
+// TODO update when available
```

Comment:
Not sure this will be necessary since it will be integrated in the SDK?

Reply:
Good point!

Reply:
Yep, very true, let's link to that instead 👍